docs/api/en/renderers/webxr/WebXRManager.html

```diff
@@ -101,6 +109,12 @@ <h3>[method:void setReferenceSpaceType]( [param:String referenceSpaceType] )</h3
 		Please check out the [link:https://developer.mozilla.org/en-US/docs/Web/API/XRReferenceSpaceType MDN] for possible values and their use cases.
 		</p>
 
+		<h3>[method:void updateCamera]( [param:PerspectiveCamera camera] )</h3>
+		<p>
+		Updates the state of the XR camera. Use this method on app level if you set [page:.cameraAutoUpdate] to *false*.
+		The method requires the non-XR camera of the scene as a parameter.
```

Comment:
It does seem important to note that the passed in cameras transformation is automatically adjusted to the position of the VR camera when calling this function.